cumulusci/cumulusci.yml

```diff
@@ -85,6 +85,20 @@ tasks:
             path: src
             revert_path: src.orig
         group: Salesforce Metadata
+    create_permission_set:
+        group: Metadata Transformations
+        description: Creates a Permission Set with specified User Permissions and assigns it to the running user.
+        class_path: cumulusci.tasks.salesforce.create_permission_sets.CreatePermissionSet
+    create_bulk_data_permission_set:
+        group: Data Operations
+        description: "Creates a Permission Set with the Hard Delete and Set Audit Fields user permissions. NOTE: the org setting to allow Set Audit Fields must be turned on."
+        class_path: cumulusci.tasks.salesforce.create_permission_sets.CreatePermissionSet
+        options:
+            api_name: CumulusCI_Bulk_Data
+            label: CumulusCI Bulk Data
```

Comment:
How about leave CumulusCI out of the name, but set a description for the permission set that explains what its for and that it was created by this task.

Reply:
Can you say more? I wanted to put CumulusCI in the name to make its purpose clear at a glance.

Reply:
Open to discussion but my thinking is: There's nothing about the permission set that is inherently tied to CumulusCI once it's in use in an org. We don't label other things that CumulusCI does in an org as CumulusCI-related just because that's how it happened to get there. I can understand wanting to provide some trail to understand how it got there, which is why I suggested putting it in the description, but including it in the developer name in particular seems heavy handed since someone may decide the permission set is also useful for non-CumulusCI-related purposes.

Reply:
Hmm. I hear you, but at the same time, this Permission Set has a somewhat odd combination of permissions - it's really useful only for performing data operations. Since it's unmanaged, the end user is free to change the API name and label at any time.

Reply:
Ok, fair enough.